rpc_test.go

```diff
@@ -543,6 +547,9 @@ func TestCtx(t *testing.T) {
 	if !serverHandler.cancelled {
 		t.Error("expected cancellation on the server side")
 	}
+	if serverHandler.connectionType != ConnectionTypeWS {
+		t.Error("wrong connection type")
```

Comment:
Thinking out loud: This go module already [depends on testify](https://github.com/filecoin-project/go-jsonrpc/blob/81c1e3fd373b68ab590d56f70eed79008d4b3e79/go.mod#L10). At some point in a separate PR we should reduce LoC and use it. For now I understand that this PR is keeping to existing style 👍

Reply:
agreed, it's very verbose as is and would be nice to do a full cleanup of these explicit `t` uses